app/services/push_notification/iss_sub_event.rb

```diff
@@ -0,0 +1,22 @@
+module PushNotification
+  module IssSubEvent
+    def payload(iss_sub:)
+      {
+        subject: {
+          subject_type: 'iss-sub',
+          iss: Rails.application.routes.url_helpers.root_url,
+          sub: iss_sub,
+        },
+      }
+    end
+
+    # Used by specs for argument matching
+    def ==(other)
+      self.class == other.class && user == other.user
+    end
+
+    def event_type
+      self.class::EVENT_TYPE
+    end
```

Comment:
Thanks for this mixin refactor! It was just an idea! not needed for this PR

But personally I think the method is simple enough, I would just leave it copy-pasted in each place, otherwise it makes for a lot of back and forth between files to tell what the result isa